src/ray/raylet/scheduling/cluster_resource_scheduler.h

```diff
@@ -148,6 +148,27 @@ class ClusterResourceScheduler : public ClusterResourceSchedulerInterface {
   ///
   ///  \return -1, if no node can schedule the current request; otherwise,
   ///          return the ID of a node that can schedule the task request.
+  int64_t GetBestSchedulableNodeLegacy(const TaskRequest &task_request,
+                                       bool actor_creation, bool force_spillback,
+                                       int64_t *violations, bool *is_infeasible);
+
+  ///  Find a node in the cluster on which we can schedule a given task request.
```

Comment:
Please reword the algorithm in simpler terms (so a reader unfamiliar with that book can understand how it is working).